openshift/4.0/templates/backup/deploy.yaml

```diff
@@ -283,33 +234,28 @@ parameters:
     required: true
     value: "1Gi"
 objects:
-  # Storage for backup.
-  - kind: PersistentVolumeClaim
+  # Configuration settings for cronjob.
+  - kind: ConfigMap
     apiVersion: v1
     metadata:
-      name: ${BACKUP_VOLUME_NAME}
+      name: ${APP_NAME}-${ROLE_NAME}
       namespace: ${PROJECT_NAMESPACE}-${ENV_NAME}
       labels:
-        name: ${BACKUP_VOLUME_NAME}
+        name: ${APP_NAME}-${ROLE_NAME}-verification
         app: ${APP_NAME}
         role: ${ROLE_NAME}
         env: ${ENV_NAME}
-    spec:
-      storageClassName: ${BACKUP_VOLUME_CLASS}
-      accessModes:
-        - ReadWriteMany
-      resources:
-        requests:
-          storage: ${BACKUP_VOLUME_SIZE}
+    data:
+      backup.conf: "0 1 * * * default ./backup.sh -s\nmssql=${DATABASE_SERVICE_NAME}:1433/${DATABASE_NAME}"
```

Comment:
Only include the relevant lines so that it doesn't parse incorrectly and add spaces.